application/src/main/java/org/opentripplanner/inspector/vector/geofencing/GeofencingZonesPropertyMapper.java

```diff
@@ -1,38 +1,35 @@
 package org.opentripplanner.inspector.vector.geofencing;
 
-import static org.opentripplanner.street.model.RentalRestrictionExtension.RestrictionType.BUSINESS_AREA_BORDER;
-import static org.opentripplanner.street.model.RentalRestrictionExtension.RestrictionType.NO_DROP_OFF;
-import static org.opentripplanner.street.model.RentalRestrictionExtension.RestrictionType.NO_TRAVERSAL;
-
+import java.util.ArrayList;
 import java.util.Collection;
-import java.util.List;
 import org.opentripplanner.apis.support.mapping.PropertyMapper;
 import org.opentripplanner.inspector.vector.KeyValue;
-import org.opentripplanner.street.model.vertex.Vertex;
+import org.opentripplanner.service.vehiclerental.model.GeofencingZone;
 
 /**
  * A {@link PropertyMapper} for the {@link GeofencingZonesLayerBuilder} for the OTP debug client.
  */
-public class GeofencingZonesPropertyMapper extends PropertyMapper<Vertex> {
+public class GeofencingZonesPropertyMapper extends PropertyMapper<GeofencingZone> {
 
   @Override
-  protected Collection<KeyValue> map(Vertex input) {
-    var ext = input.rentalRestrictions();
+  protected Collection<KeyValue> map(GeofencingZone zone) {
+    var properties = new ArrayList<KeyValue>();
+
+    properties.add(new KeyValue("id", zone.id().toString()));
+    properties.add(new KeyValue("name", zone.name().toString()));
+    properties.add(new KeyValue("network", zone.id().getFeedId()));
 
-    // this logic does a best effort attempt at a simple mapping
-    // once you have several networks on the same vertex it breaks down.
-    // for that you would really need several layers.
-    // still, for a quick visualization it is useful
-    var debug = ext.debugTypes();
-    var networks = new KeyValue("networks", String.join(",", ext.networks()));
-    if (debug.contains(BUSINESS_AREA_BORDER)) {
-      return List.of(new KeyValue("type", "business-area-border"), networks);
-    } else if (debug.contains(NO_TRAVERSAL)) {
-      return List.of(new KeyValue("type", "traversal-banned"), networks);
-    } else if (debug.contains(NO_DROP_OFF)) {
-      return List.of(new KeyValue("type", "drop-off-banned"), networks);
-    } else {
-      return List.of();
+    if (zone.isBusinessArea()) {
+      properties.add(new KeyValue("type", "business-area"));
+    } else if (zone.traversalBanned()) {
+      properties.add(new KeyValue("type", "no-traversal"));
+    } else if (zone.dropOffBanned()) {
+      properties.add(new KeyValue("type", "no-drop-off"));
     }
+
+    properties.add(new KeyValue("traversalBanned", String.valueOf(zone.traversalBanned())));
```

Comment:
You can add another static helper for converting a boolean to a string.

Reply:
These are not used actually, will remove them